frontend/src/pages/SignUp.jsx

```diff
@@ -8,17 +9,36 @@ const Auth = () => {
     fullName: '',
     email: '',
     password: '',
-    confirmPassword: ''
+    confirmPassword: '',
   });
 
   const [error, setError] = useState('');
-  const [isSignUp, setIsSignUp] = useState(true); // Toggle between SignIn and SignUp
+  const [isSignUp, setIsSignUp] = useState(true);
+  const [currentImage, setCurrentImage] = useState(0);
+
+  const images = [
+    "https://www.gvpce.ac.in/slideshow/home/Homepageslideshowphotos/2.College&Departments/18.jpg",
+    "https://www.gvpce.ac.in/slideshow/home/Homepageslideshowphotos/2.College&Departments/19.jpg",
+    "https://www.gvpce.ac.in/slideshow/home/Homepageslideshowphotos/2.College&Departments/20.jpg",
+    "https://www.gvpce.ac.in/slideshow/home/Homepageslideshowphotos/2.College&Departments/21.jpg",
+    "https://www.gvpce.ac.in/slideshow/home/Homepageslideshowphotos/2.College&Departments/22.jpg",
+    "https://www.gvpce.ac.in/slideshow/home/Homepageslideshowphotos/2.College&Departments/23.jpg",
+    // Add more images here
+  ];
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setCurrentImage((prevIndex) => (prevIndex + 1) % images.length);
+    }, 3000); // Change image every 3 seconds
+
+    return () => clearInterval(interval);
+  }, [images.length]);
```

Comment:
why useEffect is depending on images.length ?? It is not dynamic , either it can be hardcoded / fetched from db but can never be dynamic . try removing images.length from dependency array of useEffect.